requirements-prod.txt

```diff
@@ -1,7 +1,7 @@
 transformers==4.36.0
 starlette==0.27.0
 setuptools==65.5.1
-numpy==1.26.4
+numpy==1.22.2
```

Comment:
why are we downgrading?

Reply:
Previous version  was throwing error while doing indexing on production so we downgraded the numpy version